inst/tinytest/raw_data/Metamorpheus/ExperimentalDesign.tsv

```diff
@@ -0,0 +1,5 @@
+Run	Condition	BioReplicate	Fraction	Techrep

```

Comment:
First column doesn't necessarily need to be run

Reply:
We need to add validation that the join is possible, and better report the explanation (rather than a join error).

Reply:
Remove Techrep